org.eclipse.xtext.xbase.lib/src/org/eclipse/xtext/xbase/lib/ArrayExtensions.java

```diff
@@ -109,6 +111,29 @@ public static boolean equals(Object[] array, Object other) {
 		return array.equals(other);
 	}
 	
+	 /**
```

Comment:
There appears to be mixed indentation in this file

Reply:
done (I wonder what I did here)